src/common/components/build-row/index.js

```diff
@@ -44,18 +37,20 @@ const BuildRow = (props) => {
   }
 
   return (
-    <div className={ `${styles.buildRow} ${statusStyle[status]}` }>
-      <div className={ styles.item }><Link to={`/${repository.fullName}/builds/${buildId}`}>{`#${buildId}`}</Link> {statusMessage}</div>
-      <div className={ styles.item }>
+    <Row key={ buildId }>
+      <Data col="20">
+        <Link to={`/${repository.fullName}/builds/${buildId}`}>{`#${buildId}`}</Link>
+      </Data>
+      <Data col="20">
         {architecture}
-      </div>
-      <div className={ styles.item }>
+      </Data>
+      <Data col="20">
         {humanDuration}
-      </div>
-      <div className={ styles.item }>
-        {humanDateStarted}
-      </div>
-    </div>
+      </Data>
+      <Data col="40">
+        <BuildStatus link="" status={status} statusMessage={statusMessage} dateStarted={humanDateStarted} />
```

Comment:
`link` is optional, so if it's empty could be omitted.

Also BuildStatus formats `dateStarted` on its own, so you can just pass `dateStarted` to it and remove the part that formats it (it's already done in BuildStatus).